profiles/angular/skeleton/grails-app/assets/javascripts/@grails.codegen.defaultPackage.path@/core/services/DomainServiceFactory.js

```diff
@@ -0,0 +1,22 @@
+//= wrapped
+
+/*
+    NOTE: This file is used by the create-ng-domain action.
+    You can modify or extend the DomainServiceFactory but it is recommended that you not delete it.
+*/
+
+angular
+    .module("@grails.codegen.defaultPackage@.core")
+    .factory("DomainServiceFactory", DomainServiceFactory);
+
+function DomainServiceFactory($resource) {
+    return function(url, paramDefaults, actions, options) {
+        return $resource(
+            url,
+            paramDefaults || null,
+            actions || {"update": {method: "PUT"}, "list": {method: "GET", isArray: true}},
```

Comment:
For this param I would use extends. That way if the user only wants to add a method, they can keep the defaults.